Assets/Plugins/macOS/.gitattributes

```diff
@@ -1 +1,3 @@
-libEOSSDK-Mac-Shipping.dylib filter=lfs diff=lfs merge=lfs binary
\ No newline at end of file
+libEOSSDK-Mac-Shipping.dylib filter=lfs diff=lfs merge=lfs binary
+libDynamicLibraryLoaderHelper.dylib filter=lfs diff=lfs merge=lfs binary
+MicrophoneUtility_macos.dylib filter=lfs diff=lfs merge=lfs binary
```

Comment:
It's not necessary to rectify this - so not a critique, but in case it was something not previously known, the contents of this file could be simplified to be the following:
```gitattributes
*.dylib filter=lfs diff=lfs merge=lfs binary
```

Reply:
I've had trouble getting LFS to add things when assigning like this. Now that it's in LFS, maybe it'll work? I'll go change this.